### ISSUE TITLE
Unable to use Ctrl+{PageUp,PageDown} - unbound in default yml / missing documentation?

### ISSUE THREAD

**Issue by ronjouch** [jwilm/alacritty#447]
Hi,

I'm able to use alacritty with my existing tmux configuration, but some key combinations don't work.

* **Working**: `Shift+{Left, Right, Up, Down}` (which I use to switch panes), `Ctrl+B` (which I use to (un-)maximize panes, and a slew of others that work as expected.
* **Broken**: `Ctrl+{PageUp,PageDown}` don't do anything, even though I bound them in tmux to `next-window` (letting me "switch tab" like in a web browser), and they work as expected in gnome-terminal. Same for `Shift+{PageUp,PageDown}`, which I bound to tmux `previous-page` and `next-page`.

Then I'm unsure what to do at this point:

* Is this a bug?
* If not, is it missing configuration? I see in my `alacritty.yml` that the *working* key combinations are listed in `key_bindings` (e.g. `{ key: Left, mods: Shift, chars: "\x1b[1;2D"}`) and indeed my *non-working* Ctrl+PgUp/Dn are not bound.
    * So I'd like to create two such bindings for Ctrl+PgUp/Down, but I have no idea what `chars` I should bind (Shift+Left was `\x1b[1;2D`, what is Ctrl+PgUp? My googling attempts were unsuccessful as I don't know how to *name* this thing I'm looking for: what is this string? A "tty character"? a "tty sequence"?
    * To prevent questions like mine, it would be awesome to guide users to finding `chars`. What do you think of either linking to a reference table, or suggesting some interactive unix tool capturing keystrokes and returning the equivalent tty keycode? That could be part of README.md or, better, a comment in the default YAML.

Thanks.

**Comment by jwilm** [comment]
If you're on a Linux system, you can run `showkey -a` to see the sequence sent by Control + PageUp in your original terminal.

The answer in most cases is to check your terminfo. Unfortunately, it looks like Control + Page Up/Down are xterm extensions not managed by terminfo, so we just have to copy what xterm does.

It looks like you want the sequences `\x1b[6;5~` (Down) and `\x1b[5;5~` (Up).

**Comment by ronjouch** [comment]
@jwilm thanks for the fast answer :)

> If you're on a Linux system, you can run `showkey -a` to see the sequence sent by Control + PageUp in your original terminal.

Excellent! One nit: it worked for `Ctrl+PgUp/Dn` but not for `Shift+PgUp/Dn`. Nothing is outputted by `showkey -a` (I tried in both gnome-terminal and xterm, and out of tmux), I got around it starting to understand these sequences and the fact that `Shift` is the `;2` suffix.

I'm pushing a micro-pull-request one liner mentioning `showkey -a` in the default yml.
